src/main/java/org/skriptlang/skript/bukkit/spark/SparkUtils.java

```diff
@@ -0,0 +1,162 @@
+package org.skriptlang.skript.bukkit.spark;
+
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import me.lucko.spark.api.statistic.StatisticWindow.CpuUsage;
+import me.lucko.spark.api.statistic.StatisticWindow.MillisPerTick;
+import me.lucko.spark.api.statistic.StatisticWindow.TicksPerSecond;
+import me.lucko.spark.api.statistic.misc.DoubleAverageInfo;
+import me.lucko.spark.api.statistic.types.DoubleStatistic;
+import me.lucko.spark.api.statistic.types.GenericStatistic;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+
+import java.util.Arrays;
+
+/**
+ * Utility class for accessing Spark's API.
+ */
+public class SparkUtils {
+
+	private static volatile DoubleStatistic<CpuUsage> cpuProcess;
+	private static volatile DoubleStatistic<CpuUsage> cpuSystem;
+	private static volatile GenericStatistic<DoubleAverageInfo, MillisPerTick> mspt;
+	private static volatile DoubleStatistic<TicksPerSecond> tps;
+
+	@Contract(pure = true)
+	private static @NotNull Spark spark() {
+		return SparkProvider.get();
```

Comment:
But why? Just call `SparkProvider.get()`.

Reply:
Last time we tried, it was causing issues. This was the best way.